server/cluster/cluster.go

```diff
@@ -200,6 +200,13 @@ func (c *RaftCluster) Start(s Server) error {
 		return nil
 	}
 
+	defaultReplicas := c.opt.GetMaxReplicas()
+	defaultLocationLabels := c.opt.GetLocationLabels()
+	c.ruleManager, err = placement.NewRuleManager(c.storage, defaultReplicas, defaultLocationLabels)
```

Comment:
When we disable the placement rules, will it affect the start process?

Reply:
It is always loaded no matter enabled or not

Reply:
@rleungx fixed.